src/App.tsx

```diff
@@ -1,11 +1,14 @@
-import './App.scss';
+import './App.scss'
+
+import SignUp from './components/Signup'
```

Comment:
You don't have to use "./", Simply "components/Signup" Is enough

Reply:
fixed